mocca-client/src/main/java/com/paypal/mocca/client/MoccaSerializer.java

```diff
@@ -187,7 +187,9 @@ private void writeRequestVariables(final ByteArrayOutputStream requestPayload, f
                         value = variable.value;
                     }
 
-                    if (isPojo(type)) {
+                    if(isEnum(type)) {
```

Comment:
Undo this change please. it is not necessary.

Reply:
Without this check Enum will satisfy the isPojo condition(type instanceof class) and hence it will follow the code path for POJO. To avoid this we need to predetermine if its Enum type first.

Reply:
Adding Enum under NON_POJO_TYPES wont work because the check at line number 229 (NON_POJO_TYPES.contains(type))) would fail for Enum type. contains wont work for Enum. I will try adding check under isPojo method and check. Thanks!

Reply:
Removed this condition and added enum check under isPojo method.